packages/manager/modules/account-migration/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@ovh-ux/manager-account-migration",
+  "version": "0.0.1",
```

Comment:
We should avoid using 0.x version packages in production: https://semver.org/lang/fr/#spec-item-4

Reply:
Changed to 0.0.0 until release. Thanks :)